index.d.ts

```diff
@@ -0,0 +1,12 @@
+export type ClassValue = ClassArray | ClassDictionary | string | number | null | boolean;
+
+interface ClassDictionary {
+	[id: string]: any;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+declare const clsx: (...classes: ClassValue[]) => string;
+
+export = clsx;
```

Comment:
Why not normal `export default`? IMO esModuleInterop should be enabled by default.

Reply:
I agree